docs/sources/developers/http_api/apis.md

```diff
@@ -42,11 +42,27 @@ Groups organize related functionality into logical collections. For example `das
 
 These APIs will also uses semantic versioning with three stability levels:
 
-| Level | Format     | Description                                                                 | Use Case                 |
-| ----- | ---------- | --------------------------------------------------------------------------- | ------------------------ |
-| Alpha | `v1alpha1` | Early development stage. Unstable, may contain bugs, and subject to removal | For testing new features |
-| Beta  | `v1beta1`  | More stable than alpha, but may still have some changes                     | For early production use |
-| GA    | `v1`       | Generally Available. Stable with backward compatibility guarantees          | For production use       |
+| Level | Format     | Description                                                                 | Use Case                 | Enabled By Default? |
+| ----- | ---------- | --------------------------------------------------------------------------- | ------------------------ | ------------------- |
+| Alpha | `v1alpha1` | Early development stage. Unstable, may contain bugs, and subject to removal | For testing new features | No                  |
+| Beta  | `v1beta1`  | More stable than alpha, but may still have some changes                     | For early production use | No                  |
+| GA    | `v1`       | Generally Available. Stable with backward compatibility guarantees          | For production use       | Yes                 |
+
+#### Alpha
+
+Alpha versions should not be served unless explicitly enabled by a feature flag, and should be considered completely experimental and subject to major changes.
+An Alpha version may undergo breaking changes without adding an additional version, and should not be relied upon by production workflows.
```

Comment:
```suggestion
An alpha version may undergo breaking changes without adding an additional version, and should not be relied upon by production workflows. Alpha versions may be removed completely, even without being promoted to a more stable level (e.g. an experimental API may be introduced as alpha for a new feature and subsequently removed completely, in case that feature gets canceled).
```